### PR TITLE
[JENKINS-34564] Adjusted integration test to match modified behavior

### PR DESCRIPTION
**Pull request by jglick**
Downstream of https://github.com/jenkinsci/branch-api-plugin/pull/50.

@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

**Comment by svanoort** [comment]
🐝

**Comment by svanoort** [comment]
LGTM with the new changes, one the merge conflict is resolved

File: pom.xml [hunk 1]
```diff
@@ -204,6 +204,12 @@ THE SOFTWARE.
             <version>1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.1.4</version>
+            <scope>test</scope>
```
Comment (KostyaSha) [inline pom.xml]: because old 1.x is used in core?

**Pull request merged**